diff --git a/lib/aurora.cpp b/lib/aurora.cpp
index 8cc5e4c..31cbf5d 100644
--- a/lib/aurora.cpp
+++ b/lib/aurora.cpp
@@ -1185,7 +1185,8 @@ void encode_presentation_snapshot(const wgpu::CommandEncoder& encoder,
         wgpu::RenderPassColorAttachment{
             .view = image.texture.view,
             .loadOp = wgpu::LoadOp::Clear,
             .storeOp = wgpu::StoreOp::Store,
+            .clearValue = {.r = 0.0, .g = 0.0, .b = 0.0, .a = 1.0},
         },
     };
     const wgpu::RenderPassDescriptor renderPassDescriptor{
