diff --git a/lib/gfx/common.cpp b/lib/gfx/common.cpp
--- a/lib/gfx/common.cpp
+++ b/lib/gfx/common.cpp
@@ -274,6 +274,7 @@ static TextureHandle new_resolve_source_snapshot(wgpu::Extent3D size, wgpu::Tex
   const wgpu::TextureDescriptor textureDescriptor{
       .label = "GX Copy Source Snapshot",
-      .usage = wgpu::TextureUsage::TextureBinding | wgpu::TextureUsage::CopyDst,
+      .usage = wgpu::TextureUsage::TextureBinding | wgpu::TextureUsage::CopySrc |
+               wgpu::TextureUsage::CopyDst,
       .dimension = wgpu::TextureDimension::e2D,
       .size = size,
       .format = format,
